src/taipy/templates/multi-page-gui-core/{{cookiecutter.application_name}}/pages/root/root.md

```diff
@@ -0,0 +1,13 @@
+<|navbar|>
+
+<|layout|columns=1 4|
+
+<|{scenario}|scenario_selector|>
+
+<|part|class_name=main|{is_scenario_selected}|
```

Comment:
"{is_scenario_selected}" seems useless: reflects "scenario"

Reply:
I'm not sure to understand what you meant here. Can you suggest a change?